tabs/cli.js

```diff
@@ -174,12 +174,10 @@ TABS.cli.read = function (readInfo) {
                 } else {
 
                     GUI.timeout_add('waiting_for_bootup', function waiting_for_bootup() {
-                        MSP.send_message(MSP_codes.MSP_IDENT, false, false, function () {
```

Comment:
This and the below uses of `MSP_IDENT` are there to make sure the flight controller is ready before the code in the callback is executed. Instead of removing them, we should replace the call to `MSP_IDENT` with a call to something like `MSP_STATUS` / `MSP_STATUS_EX`.